SofaKernel/framework/sofa/core/VecId.h

```diff
@@ -150,8 +151,14 @@ class TStandardVec<V_MATDERIV, vaccess>
 public:
     typedef TVecId<V_MATDERIV, vaccess> MyVecId;
 
-    static MyVecId holonomicC()    { return MyVecId(1);}
-    static MyVecId nonHolonomicC() { return MyVecId(2);}
+    static MyVecId constraintMatrix()    { return MyVecId(1);} // jacobian matrix of constraints
```

Comment:
Yes we can rename constraintMatrix by constraintJacobian it is perhaps more clear.
In this case, we could name the second mappingJacobian (to be coherent).